bin/brew

```diff
@@ -108,14 +108,14 @@ HOMEBREW_LIBRARY="${HOMEBREW_REPOSITORY}/Library"
 
 # Load Homebrew's variable configuration files from disk.
 export_homebrew_env_file() {
-  local env_file
+  local env_file="${1}"
+  local homebrew_env_regex="^HOMEBREW_[A-Z_]+=[^=]*$"
```

Comment:
Ideally we should be able to reject spaces after the `=`, but we have a lot of env variables that take multiple values that are space-separated.

Personally I'd be in favour of deprecating separating these with spaces and delimit them with `,` or `;` instead.

Reply:
I'd rather we did't deprecate for this. An alternative approach would be to "build" the line separately by extracting first the `HOMEBREW_[A-Z_]+`, then the `=`, then the rest of the line without `=` or `HOMEBREW_` in it.